k8s/k8s.go

```diff
@@ -279,6 +295,18 @@ func CmdAddK8s(ctx context.Context, args *skel.CmdArgs, conf types.NetConf, epID
 			return nil, errors.New("IPAM plugin returned missing IP config")
 		}
 
+		if conf.IPAM.Type == "azure-vnet-ipam" {
+			// Store off the subnet information - we need to pass this back on each subsequent call
+			// to the Azure IPAM plugin.
+			subnetPrefix := result.IPs[0].Address
+			subnetPrefix.IP = subnetPrefix.IP.Mask(subnetPrefix.Mask)
+			fn := fmt.Sprintf(azureFilenameTemplate, conf.Name)
+			if err := ioutil.WriteFile(fn, []byte(subnetPrefix.String()), 0644); err != nil {
+				return nil, err
+			}
+			logger.Infof("Stored azure subnet on disk: %s", subnetPrefix)
+		}
+
 	case ipAddrs != "" && ipAddrsNoIpam != "":
```

Comment:
Worth erroring-out if the user tries to mix Azure IPAM with incompatible options?